### ISSUE TITLE
Some help with the 3rd party settings

### ISSUE THREAD

**Issue by ZuluPilgrim** [home-assistant/home-assistant.github.io#28171]
### Feedback

The documents says enable plugin Discovery in OctoPrint but I have no idea where to find that and even OctoPrint does not seem to document it well.

While not the responsibility of HASS, this info would make it easier for the user.

### URL

https://www.home-assistant.io/integrations/octoprint/

### Version

2023.7.1

**Comment by ZuluPilgrim** [comment]
I have dug further into OctoPrint and there is a plugin called "Discovery". If this is what is meant then maybe reword as "ensure that the Discovery plugin" is enabled.

**Comment by klaasnicolaas** [comment]
Feel free to create a pull request and change the wording 😉

**Comment by ZuluPilgrim** [comment]
sure... not sure what that means but will Google it :)

André Basel
027 640 2527

Sent from ProtonMail mobile

-------- Original Message --------
